rgeomstats/vectorization.R

```diff
@@ -0,0 +1,16 @@
+ToNdarray <- function(x, to.ndim, axis = 0){
+  CurrentNdim <- length(dim(x))
+  if(CurrentNdim == to.ndim - 1){
```

Comment:
Style: use spacings around the if conditions, as in:
https://google.github.io/styleguide/Rguide.xml#else